lib/bindings/MQTTBinding.js

```diff
@@ -123,21 +123,23 @@ function unsubscribeAll(callback) {
  * Start the binding.
  */
 function start(callback) {
+    let configuration = config.getConfig().mqtt;
     var options = {
+        protocol: configuration.protocol != null ? configuration.protocol : "mqtt",
```

Comment:
New parameters should:

* Be included at https://github.com/telefonicaid/iotagent-json/tree/master/docs/installationguide.md#mqtt-configuration
* Have an associated environment variable (IOTA_MQTT_xxxx), documented also in that file and properly treated in the code.